easybuild/easyconfigs/w/waLBerla/waLBerla-6.1-codegen-foss-2023b.eb

```diff
@@ -0,0 +1,85 @@
+# easyconfig file for waLBerla v6.1
+easyblock = 'CMakeMakeCp'
+
+name = 'waLBerla'
+version = '6.1'
+versionsuffix = '-codegen'
+
+homepage = "https://walberla.net/index.html"
+description = """Widely applicable Lattics-Boltzmann from Erlangen is a
+block-structured high-performance framework for multiphysics simulations"""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+source_urls = ['https://zenodo.org/records/10054460/files/']
+sources = ['%(namelower)s-v%(version)s.zip']
+checksums = ['b1599a24e77f2d11883fcef06861684320fdde4add62d57578bd9542882b9ac5']
+patches = ['waLBerla-6.1-codegen-MirroredStencilDirections.patch', 'waLBerla-6.1-codegen-fix-missing-cstint.patch'] 
+
+
+builddependencies = [('CMake', '3.27.6')]
+
+dependencies = [
+    ('Boost.MPI', '1.83.0'),
+    ('pybind11', '2.11.1'),
+    ('Python', '3.11.5'),
+    ('pystencils', '1.3.4'),
+    ('lbmpy', '1.3.7'),
+]
+	
+exts_defaultclass = 'PythonPackage'
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'download_dep_fail': True,
+    'sanity_pip_check': True,
+}
+
+exts_list = [
+    ('setuptools', '80.9.0', {
+        'checksums': ['f36b47402ecde768dbfafc46e8e4207b4360c654f1f3bb84475f0a28628fb19c'],
+    }),
+    ('jinja2', '3.1.6', {
+        'checksums': ['0137fb05990d35f1275a587e9aee6d56da821fc83491a0fb838183be43f66d6d'],
+    }),
+    ('py-cpuinfo', '9.0.0', {
+        'checksums': ['3cdbbf3fac90dc6f118bfd64384f309edeadd902d7c8fb17f02ffa1fc3f49690'],
+    }),
+    ('numpy', '2.3.0', {
+        'checksums': ['581f87f9e9e9db2cba2141400e160e9dd644ee248788d6f90636eeb8fd9260a6'],
+    }),
+]
+max_parallel = 1
+
+configopts = "-DWALBERLA_BUILD_WITH_PYTHON=ON "
+configopts += "-DWALBERLA_BUILD_SHOWCASES=OFF "
+configopts += "-DWALBERLA_BUILD_WITH_CODEGEN=ON "
+configopts += "-DWALBERLA_BUILD_DOC=OFF "
+configopts += "-DWALBERLA_BUILD_BENCHMARKS=OFF"
+configopts += "-DPython_ROOT_DIR=$EBROOTPYTHON"
+
+install_cmd = 'make pythonModuleInstall'
+
+files_to_copy = [
+    (['%(builddir)s/easybuild_obj/*'], 'build'),
+    (['%(start_dir)s/src/*'], 'src'),
+]
+
+sanity_check_paths = {
+    'files': [
+        'build/src/waLBerlaDefinitions.h',
+        'build/apps/tutorials/basics/01_Tutorial_BlocksAndFields1',
+        'build/utilities/filterCompileCommands.py'
+    ],
+    'dirs': ['src', 'build/apps', 'build/tests', 'build/utilities']
+}
+
+sanity_check_commands = [
+    "mkdir -p %(builddir)s && cp -a %(installdir)s/build/apps/tutorials/lbm %(builddir)s/",
+    "cd %(builddir)s/lbm && chmod -R u+w . && ./01_BasicLBM 01_BasicLBM.prm",
+]
+
+sanity_check_commands = ["python3 -c 'import walberla'"]
```

Comment:
This just overwrites `sanity_check_commands` in line 77?

Reply:
Yes, sorry, that was an old version of the easyconfig, I thought I had pushed changes earlier. I've fixed it now.